pkg/apiclient/_.primary.swagger.json

```diff
@@ -31,12 +31,6 @@
   "definitions": {
     "io.k8s.apimachinery.pkg.runtime.Object": {
       "title": "This is a hack do deal with this problem: https://github.com/kubernetes/kube-openapi/issues/174"
-    },
-    "github.com.argoproj.argo.pkg.apis.workflow.v1alpha1.ParallelSteps": {
```

Comment:
I guess I should have added a comment describing why this was necessary.

@sarabala1979 why did you find it necessary to remove this section?

Reply:
My `git merge` messed up. I fixed it